README.md

```diff
@@ -95,10 +108,7 @@ be human-browsable.
     - `walletType`: The wallet type, either 'API3' or 'Provider'
     - `address`: The target address, eg. "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC"
 - `signedKeeperConditions`:
-  - `deviationFactorThreshold`: The factor by which to multiply the provider's deviation threshold to calculate API3's
-    `_conditionParameters`
-  - `ttlMinutes`: The number of minutes that need to have elapsed since the on-chain value exceeded API3's
```

Comment:
So we're getting rid of this, right? This is what you talked about on yesterday's call, right?

Reply:
Hey, that's correct, I clearly misunderstood the requirement.
I've removed the TTL and I've switched the Signed Data Keeper update threshold to an absolute percentage rather than a relative percentage (relative to the provider's threshold).

Reply:
I will ask Burak to confirm, but during the call yesterday I was lead to believe that this was not required?
I came up with the `ttlMinutes` field, Burak has always been against it.

Reply:
We want two independent requirements:
1. Deviation threshold
2. Freshness threshold, updates no matter what if the on-chain value is outdated by X seconds

Reply:
Thanks @bbenligiray I will partially revert the specific changes. I thought you were against the TTL field :smile:

Reply:
I am against it as you first described it. The two thresholds should be fully independent.

Reply:
I have replaced this with
```
  - `updateConditionPercentage`: API3's update condition percentage, eg: 0.1 for 0.1%
  - `ttlSeconds`: Time in seconds since the last update, past which an update will be forced
```